db-tests/src/test/java/gov/nasa/jpl/aerie/database/PlanCollaborationTests.java

```diff
@@ -2030,12 +2029,14 @@ void modifyAndDeletesApplyCorrectly() throws SQLException {
       final int deleteUncontestedActId = insertActivity(basePlan);
       final int modifyContestedSupplyingActId = insertActivity(basePlan);
       final int modifyContestedReceivingActId = insertActivity(basePlan);
-      final int deleteContestedSupplyingActId = insertActivity(basePlan);
-      final int deleteContestedReceivingActId = insertActivity(basePlan);
+      final int deleteContestedSupplyingActIdResolveSupplying = insertActivity(basePlan);
```

Comment:
Minor refactor: Can these variables be renamed so that `ActId` is at the end (ie: `deleteContestedSupplyingResolveSupplyingActId`)?

Reply:
Makes sense! Nouns come after adjectives in English 🤦 Done 👍